scripts/install_dict.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python3
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2017 Michal Siedlaczek <michal.siedlaczek@gmail.com>
+
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""A script installing Hunspell dictionaries.
+
+Use: python -m scripts.install_dict [--list] [lang [lang [...]]]
+"""
+
+import argparse
+import base64
+import json
+import os
+import sys
+import re
+import urllib.parse
+import urllib.request
+import attr
+
```

Comment:
Add a `sys.path.insert(0, os.path.join(os.path.dirname(__file__), os.pardir))` here too so this can be invoked like `python3 scripts/install_dict.py` as well.